### PR TITLE
Vulkan: set ColorBlendState info (used by logicOp)

### PR DESCRIPTION
**Pull request by raven02**
We have colorblendstate used in LogicOp

	if (rsx::method_registers.logic_op_enabled())
	{
		properties.cs.logicOpEnable = true;
		properties.cs.logicOp = vk::get_logic_op(rsx::method_registers.logic_operation());
	}

### CODE REVIEW

**Comment by raven02** [comment]
@kd-11 , looks good to you?

File: rpcs3/Emu/RSX/VK/VKGSRender.cpp [hunk 1]
```diff
@@ -923,6 +923,9 @@ bool VKGSRender::load_program()
 		}
 	}
 
+	properties.cs.attachmentCount = 1;
```
Comment (vlj) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: Shouldn't it be m_draw_buffers_count ? I think it's the size of att_state array used during the draw call.

Comment (raven02) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: Alright let me check this out as i remember debug layer complaint about the attachment count

Comment (raven02) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: Confirmed set attachmentCount to m_draw_buffers_count fixed the debug error . I will merge this one .

File: rpcs3/Emu/RSX/VK/VKGSRender.cpp [hunk 2]
```diff
@@ -926,6 +926,9 @@ bool VKGSRender::load_program()
 		}
 	}
 
+	properties.cs.attachmentCount = m_draw_buffers_count;
```
Comment (kd-11) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: You also need to set the sType field. Other than that, it should be good to go.

Comment (raven02) [inline rpcs3/Emu/RSX/VK/VKGSRender.cpp]: Alright i will set it as well .

**Pull request merged**